File: src/main/java/com/cloudpi/cloudpi_backend/files/structure/repositories/FileRepository.java
package com.cloudpi.cloudpi_backend.files.structure.repositories;

import com.cloudpi.cloudpi_backend.files.structure.entities.FileEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
interface FileRepository extends JpaRepository<FileEntity, Long> {

}

File: src/main/java/com/cloudpi/cloudpi_backend/files_info/repositories/FileRepository.kt
package com.cloudpi.cloudpi_backend.files_info.repositories

import com.cloudpi.cloudpi_backend.files_info.entities.kotlin.FileEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface FileRepository : JpaRepository<FileEntity, Long>